code/modules/ruins/lavalandruin_code/techcult.dm

```diff
@@ -0,0 +1,592 @@
+	desc = "An ID card used by leader of the religious group praising misterious machine god."
+	access = list(ACCESS_MECHANICUS_BASIC, ACCESS_MECHANICUS_LEADER, ACCESS_ROBOTICS)
+
+/***************** Closets *****************/
+
+/obj/structure/closet/secure_closet/mechanicus
+	name = "tech storage"
+	req_access = list(ACCESS_MECHANICUS_BASIC)
+
+/obj/structure/closet/secure_closet/mechanicus/implants
+	name = "implants storage"
+
+/obj/structure/closet/secure_closet/mechanicus/implants/PopulateContents()
+	..()
+	var/static/items_inside = list(
+		/obj/item/organ/cyberimp/arm/toolset = 1,
+		/obj/item/organ/cyberimp/arm/surgery = 1,
+		/obj/item/organ/cyberimp/chest/reviver = 3,
+		/obj/item/organ/cyberimp/chest/nutriment/plus = 3,
+		/obj/item/organ/tongue/robot = 3,
+		/obj/item/organ/lungs/cybernetic/tier3 = 3,
+		/obj/item/organ/heart/cybernetic/tier3 = 3)
+	generate_items_inside(items_inside,src)
+
+/obj/structure/closet/secure_closet/mechanicus/augs
+	name = "augmentation storage"
+
+/obj/structure/closet/secure_closet/mechanicus/augs/PopulateContents()
+	..()
+	var/static/items_inside = list(
+		/obj/item/bodypart/chest/robot = 2,
+		/obj/item/bodypart/head/robot = 2,
+		/obj/item/bodypart/l_arm/robot = 2,
+		/obj/item/bodypart/r_arm/robot = 2,
+		/obj/item/bodypart/l_leg/robot = 2,
+		/obj/item/bodypart/r_leg/robot = 3)
+	generate_items_inside(items_inside,src)
+
+/***************** Armor *****************/
+
+//Basic
+/obj/item/clothing/suit/hooded/techpriest/armor
+	name = "armored techpriest robes"
+	desc = "An armored version of robes worn by followers of the machine god."
+	allowed = list(/obj/item/flashlight, /obj/item/tank/internals, /obj/item/pickaxe, /obj/item/organ/regenerative_core/legion, /obj/item/kitchen/knife/combat/bone, /obj/item/kitchen/knife/combat/survival)
+	armor = list("melee" = 45, "bullet" = 10, "laser" = 25, "energy" = 25, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 60, "acid" = 60)
+	hoodtype = /obj/item/clothing/head/hooded/techpriest/armor
+
+/obj/item/clothing/head/hooded/techpriest/armor
+	name = "armored techpriest's hood"
+	desc = "An armored version of hood worn by followers of the machine god."
+	armor = list("melee" = 45, "bullet" = 10, "laser" = 25, "energy" = 25, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 60, "acid" = 60)
+
+//Leader
+//Don't yell at me, it's super balanced, and in world of Lavaland everything is overpowered anyway.
+/obj/item/clothing/suit/hooded/techpriest/armor/lead
+	name = "blessed tech robes"
+	desc = "From the rage of the beast, Machine God protect us."
+	armor = list("melee" = 65, "bullet" = 40, "laser" = 40, "energy" = 40, "bomb" = 40, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 90)
+	hoodtype = /obj/item/clothing/head/hooded/techpriest/armor/lead
+
+/obj/item/clothing/suit/hooded/techpriest/armor/lead/Initialize()
+	. = ..()
+	AddComponent(/datum/component/anti_magic, FALSE, FALSE, TRUE, ITEM_SLOT_OCLOTHING)
+
+/obj/item/clothing/head/hooded/techpriest/armor/lead
```

Comment:
er, this has the exact same typepath as the robes

Reply:
Nope, it's perfectly fine.
"clothing/head", not "clothing/suit"